thirdeye/thirdeye-frontend/app/pods/components/self-serve-alert-yaml-details/template.hbs

```diff
@@ -1,80 +1,104 @@
-    <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.createdBy}}>
-      {{if alertData.createdBy alertData.createdBy 'N/A'}}</dd>
-
-    <dt class="col-md-4">Updated By</dt>
-    <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.updatedBy}}>
-      {{if alertData.updatedBy alertData.updatedBy 'N/A'}}</dd>
-
-    <dt class="col-md-4">Last detection</dt>
-    <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.lastDetectionTime}}>
-      {{if alertData.lastDetectionTime alertData.lastDetectionTime 'N/A'}}</dd>
-
-    <dt class="col-md-4">Subscribed Groups</dt>
-    <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{subscribedGroups}}>
-      {{if subscribedGroups subscribedGroups 'N/A'}}
-    </dd>
-  </dl>
-</div>
-{{/if}}
+  <div class="te-alert-detail-breakdown row">
+    <dl class="col-xs-12 col-sm-5 row">
+      <dt class="col-md-4">
+        Metric
+      </dt>
+      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.metric}}>
+        {{if alertData.metric alertData.metric "N/A"}}
+      </dd>
+      <dt class="col-md-4">
+        Dataset
+      </dt>
+      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.dataset}}>
+        {{if alertData.dataset alertData.dataset "N/A"}}
+      </dd>
+      <dt class="col-md-4">
+        Filtered By
+      </dt>
+      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.filters}}>
+        {{if alertData.filters alertData.filters "N/A"}}
+      </dd>
+      <dt class="col-md-4">
+        Breakdown By
+      </dt>
+      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.dimensionExploration}}>
+        {{if alertData.dimensionExploration alertData.dimensionExploration "N/A"}}
+      </dd>
+    </dl>
+    <dl class="col-xs-12 col-sm-5 row">
+      <dt class="col-md-4">
+        Created By
+      </dt>
+      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.createdBy}}>
+        {{if alertData.createdBy alertData.createdBy "N/A"}}
+      </dd>
+      <dt class="col-md-4">
+        Updated By
+      </dt>
+      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.updatedBy}}>
+        {{if alertData.updatedBy alertData.updatedBy "N/A"}}
+      </dd>
+      <dt class="col-md-4">
+        Last detection
+      </dt>
+      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.lastDetectionTime}}>
+        {{if alertData.lastDetectionTime alertData.lastDetectionTime "N/A"}}
+      </dd>
+      <dt class="col-md-4">
+        Subscribed Groups
+      </dt>
+      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{subscribedGroups}}>
+        {{if subscribedGroups subscribedGroups "N/A"}}
+      </dd>
+    </dl>
+  </div>
+{{/if}}
```

Comment:
I realize this may be beyond the scope, but...   could we do something like:  

```
{{#each this.alertDataArray as |data|}}
   <dt class="col-md-4">
        {{data.name}}
      </dt>
      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.lastDetectionTime}}>
        {{if data.value data.value  "N/A"}}
      </dd>
  {{/each}}
```

where **_alertDataArray_** is and array build inside the component/controller?  

if we want to be even more efficient... how about creating a small component 

Again - I realize it's way beyond the scope, so feel free to skip. I just wanted to bring it up so we can keep it in mind as we refactor the codebase

Reply:
Yes, I agree. I haven't check all logic in `self-serve-alert-yaml-details`. This PR (ticket) is to fix self-serve-alert-detail component. I don't want to fix multiple issues unrelated to this UI issue in one PR. Please create another ticket. We can refactor step by step.